src/main/java/com/icthh/xm/ms/entity/service/XmEntitySpecService.java

```diff
@@ -35,21 +36,17 @@
 import org.springframework.stereotype.Service;
 import org.springframework.util.AntPathMatcher;
 
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
please avoid wildcard and use explicit imports

Reply:
replaced